command/sub8_missions/missions/pose_sequence_test.py

```diff
@@ -0,0 +1,15 @@
+from txros import util
+import numpy as np
+from sub8.sub_singleton import PoseSequenceCommander
+
+
+@util.cancellableInlineCallbacks
+def run(sub_singleton):
+    print "Starting"
+    poses = [np.array([-1, 0, 0, 0, 0, 3]),
+    		np.array([-1, 0, 0, 0, 0, 0]),
+    		np.array([-1, 0, 0, 0, 0, 3]),]
+    poses_command = PoseSequenceCommander(sub_singleton,poses)
```

Comment:
We tend to represent poses as 3-vectors concatenated with quaternions. Can you make it look like this?:
```python
PoseSequenceCommander(sub_singleton,positions,orientations)
```
 Alternatively you could concatenate them but you 
should raise an Exception with a helpful error msg if it is not of length 7.